Carta.py

```diff
@@ -365,6 +427,9 @@ def decideWinner(self):
             self.GPinfo.youWin = False
             self.GPinfo.opponentWin = True
             statement = "Carta.py: decideWinner: No available GrabCard, opponent win"
```

Comment:
modify
`checkGrabbingAvailable(...) is False`

to

`self.GPinfo.currentGrabCardStatus = GrabCardStatus.INVALID`

After this change, remove `checkGrabbingAvailable` function